src/lib/state/utils.ts

```
@@ -0,0 +1,56 @@
+  [key: string]: any;
+};
+export const DEFAULT_LIMIT = 10;
+
+export function getQueryKey<T extends KeyParams>(key: string, params?: T) {
+  return [key, ...(params ? [params] : [])];
+}
+
+// for infinite query pages  to flatList data
+export function normalizePages<T>(pages?: PaginateQuery<T>[]): T[] {
+  return pages
+    ? pages.reduce((prev: T[], current) => [...prev, ...current.results], [])
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)